src/extensions/amazon-sales/release-notes/index.md

```diff
@@ -21,6 +21,32 @@ The release notes include:
 
 See [Upcoming Releases](https://devdocs.magento.com/release/) for versioning, support, and compatibility.
 
+### v4.2.0
+
+Amazon Sales Channel 4.2.0 is compatible with versions 2.3.x but only supported in versions 2.4.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. If you have a previous Amazon Sales Channel version installed and attempt to update your Magento to version 2.4.0, you will be prompted to update the extension before you can complete the Magento update.
+
+This version of Amazon Sales Channel includes a new feature along with improvements and fixes.
+
+-  {:.new}Amazon Sales Channel has been enhanced to accept text-based address data and match it to standardized address formats, including city, state, and zip code. This enables order and shipping data to synchronize with Amazon without address errors. Example: Shopper inputs the city, state, zip code as `Escondido, californiA 92025-1501`. Amazon Sales Channel imports and matches the data to the standard format as `Escondido, CA 92025`, and then syncs it back to Amazon in this standardized format.
+
+-  {:.new}Support for PHP 7.4 has been added.
+
+-  {:.fix}<!--CHAN-4334-->The extension has been updated for support with Magento 2.4.x. Previous versions may be compatible with Magento 2.4.x, but are not supported. See [Upcoming releases](https://devdocs.magento.com/release/) for version compatibility. Amazon Sales Channel must be updated to 4.2.0 before the Magento 2.4.0 update can be completed.
+
+-  {:.fix}<!--CHAN-4431-->Corrected an error causing an "Access Denied" error for UK customers.
+
+-  {:.fix}<!--CHAN-4394-->Corrected an issue preventing the Amazon shipping status from syncing to the corresponding Magento order, thus “locking” the order’s shipping status as `Pending` in Magento and `Unshipped` in Amazon. With the implementation of the new standardized address feature, these shipping status errors have been resolved.
+
+-  {:.fix}<!--ticket#-->Order synchronization has been updated to ignore failed order imports, thus reducing multiple sync attempts and allowing subsequent imports to process, with order sync requests submitted every 5 minutes. Synchronization errors are still recorded in the error log. Synchronization errors are logged, but marked as "processed" to allow further logging functions. As part of this fix, Amazon Sales Channel now automatically removes excess data collected for orders that fail to create in Magento.
+
+-  {:.fix}Error logging has been updated to collect more information for “Uncaught Exception” and extension update errors.
+
+-  {:.fix}<!--ticket#-->Corrected an issue causing the initial sync of "Lowest Price" data to fail due to missing "Price" value.
+
+-  {:.fix}<!--CHAN-4410-->Corrected issues causing errors when the date range field is left blank when filtering your orders view.
```

Comment:
```suggestion
-  {:.fix}<!--CHAN-4410-->Corrected issues which caused errors in blank date range fields when filtering the Orders view.
```

Reply:
Is Orders view the name of an actual view int he Admin? If so, it should be capped as suggested. If not, just orders view.

Reply:
the errors were not in the date range field.   the errors occurred because the date range field is left blank.  it is the Amazon orders screen.   reworded as: Corrected issues that caused filtering errors in the _Amazon orders_ view when the date range field is left blank.